docs/edge.md

```diff
@@ -108,19 +112,24 @@ How often should Edge run in seconds? <br/>
 
 #### calculate_since_number_of_days
 Number of days of data against which Edge calculates Win Rate, Risk Reward and Expectancy
-Note that it downloads historical data so increasing this number would lead to slowing down the bot<br/>
+Note that it downloads historical data so increasing this number would lead to slowing down the bot.<br/>
```

Comment:
This is technically untrue - yes, startuptime would be slowed down by downloading more data (if it's not available) - but the slowdown is while running edge, not because of the data download ... 

(nothing blocking though ...) just saying